### DISCUSSION TITLE
Problem with cmdVel().

### DISCUSSION THREAD

**Discussion by Wallat1** [bitcraze/discussions#304] [Q&A]
Hi.

I would like to know if do you have some example using the command cmdVel(roll, pitch, yawrate, thrust)  from crazyswarm Python API?, the motive of my question is because I can not get the above command to work as the command  send_setpoint(roll, pitch, yawrate, thrust) in ramp.py. 

I attach my program as visual aid:
![20220919_154351](https://user-images.githubusercontent.com/112857979/191158513-cc3ce627-8ad4-4e0d-ad46-5bf9afa37a28.jpg)

I know that the question should be done in the crazyswarm forum but their response time is not very fast.

Regards.

**Comment by knmcguire** [comment]
For context, this is the crazyswarm2 discussion: https://github.com/USC-ACTLab/crazyswarm/discussions/683
@Wallat1 I'll answer you there because that will be better in maintainability

**Reply by Wallat1** [reply]
Hi.

Thank you for your help.

Regards.
